app/models/user.rb

```diff
@@ -40,7 +40,7 @@ class User < CouchRest::Model::Base
     :mx_with_fallback => true
 
 
-  validates_with InviteCodeValidator, on: :create
+  validates_with InviteCodeValidator, on: :create, if: -> {APP_CONFIG[:invite_required]}
```

Comment:
This toggles the InviteCodeValidator on and off. I had to look at the [documentation for `validates_with`](https://devdocs.io/rails/activemodel/validations/classmethods#method-i-validates_with) to piece this together.

Can you tell what the `-> { }` does and if/why its needed here?